whynotearth/BrowTricks
# 174 add spinner while loading

# Issue Being Addressed

#174 

# Type of PR

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For Bug Fixes:** What was the root cause? How do your changes fix the bug effectively?

**For Refactors:** What are we refactoring and why? How do your changes address this need for refactoring?

**For New Features:** What feature are we adding? Explain your technical approach to doing this.  
Add spinner in client page  
Add fade animation between route changes in client page (WithTitleBarLayout)

**For Feature Updates:** What feature are we updating? Explain your technical approach to doing this.

**For Other:** State what 'type' of PR yours is and elaborate on the PR's technical content.

# How to Test/Reproduce

Check client edit page, go to sub items of client page and return to client edit.


--------

I know the image and animation of spinner is not pretty right now, waiting for @bamboochow to send a better PNG or SVG image and probably explain more about animation.
